app/src/debug/res/values-cs/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What happened to the "product" attribute ?

Reply:
I see. I'd prefer not to complicate the translation process if possible. This is intended to add the " (Debug)" string in the name of the app, which is shown in several places.
Do you think the app name could be defined programmatically instead ? eg. define `app_name`, which is translated as usual and `debug_suffix`, which is either "" or " (Debug)" and set the app name to the concatenation of the two.

Reply:
It would be possible to define two different app names per strings.xml file, e.g. `app_name_debug` and `app_name_release` and Gradle can replace `app_name` with one of those two depending on the current build type. From my testing this didn't work with one resource just named `app_name`, so they could be named `app_name_debug` and `app_name_release` or `app_name_default` or something like that. This would be easier I think.